src/pages/Synths/Home/SynthsSection/SynthsTable/SynthsTable.tsx

```diff
@@ -102,21 +102,15 @@ export const SynthsTable: FC<SynthsTableProps> = memo(
 							{
 								Header: <>{t('synths.home.table.24h-change-col')}</>,
 								sortType: 'basic',
+								accessor: (originalRow: any) =>
```

Comment:
Since react-table isn't 100% typed correctly, I'm using `any` here for now.

Reply:
why is it not typed correctly?

Reply:
you could also destructure originalRow here maybe just do `{ historicalRates }`

Reply:
its on my todo list... react-table v7 typing is quite complex with all the different plugins, etc...